session_0/word_processing.rb

```diff
@@ -34,11 +37,33 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+	# l is the length of the shorter word,
+	l=word_1.size > word_2.size ? word_2.size : word_1.size
+	max=0
+	for i in 1..l
```

Comment:
Do you think we can use `find` function here? https://apidock.com/ruby/Enumerable/find